mobify/platform-scaffold
# AMP-45: main nav

Finalize AMP Main Navigation

 **JIRA**: https://mobify.atlassian.net/browse/AMP-45

## Changes
- Add `<HeaderBar>` to Main Navigation Sidebar
- Update Sign In List Tile to match PWA 

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](http://localhost:3000/)
- Open Main Navigation and make sure it looks same as PWA Main Navigation
